.coafile

```diff
@@ -0,0 +1,8 @@
+[Default]
+enabled = True
+
+[commit]
+bears = GitCommitBear
+
+[python]
+bears = PEP8Bear
```

Comment:
Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.